.github/workflows/android-device-test.yml

```diff
@@ -141,7 +129,7 @@ jobs:
           testSpecArn: ${{ matrix.test.testSpecArn }}
 
       - uses: LouisBrunner/checks-action@v2.0.0
-        if: always() && env.run_device_test == 'true'
+        if: always() && (env.run_device_test == 'true' || matrix.test.name == 'Android Render Tests')
```

Comment:
This file seems a bit long for yaml... 100+ lines long... I would consider moving the invocation to a separate script files and keep the minimum possible here...

Reply:
I don't really see any opportunities to make it shorter. When you move stuff into a script you still need to pass a lot of variables, and this has the potential to make it even longer.

The matrix takes up a lot of space, but this is also not something you can move into another file.